modules/fedimint-wallet-server/src/envs.rs

```diff
@@ -0,0 +1,23 @@
+use fedimint_core::util::FmtCompact as _;
+use fedimint_logging::LOG_MODULE_WALLET;
+use fedimint_wallet_common::FEERATE_MULTIPLIER_DEFAULT;
+use tracing::warn;
+
+pub const FM_WALLET_FEERATE_MULTIPLIER_ENV: &str = "FM_WALLET_FEERATE_MULTIPLIER";
+
+pub fn get_feerate_multiplier() -> f64 {
+    if let Ok(mult) = std::env::var(FM_WALLET_FEERATE_MULTIPLIER_ENV) {
+        match mult.parse::<f64>() {
+            Ok(mult) => return mult.clamp(0.1, 32.0),
```

Comment:
I think it's too risky to allow a federation to set a 0.1 feerate multiplier. Let's bump the min to 1.0.